pixi/src/ui/PageExperience.js

```diff
@@ -42,16 +45,25 @@ export default class PageExperience {
     } else {
       this.toggleLoading(true);
 
+      // Core Web Vitals
       const check = new PageExperienceCheck();
       const report = await check.run(inputUrl);
-
       for (const [id, metric] of Object.entries(
         report.coreWebVitals.fieldData
       )) {
         this.reportViews[id] =
           this.reportViews[id] || new CoreWebVitalsReport(document, id);
         this.reportViews[id].render(metric);
       }
+
+      // Additional checks
+      const safeBrowsingCheck = new SafeBrowsingCheck();
```

Comment:
In this case, if it takes some time to retrieve the core web vitals data, will the safe browsing result be able to display if ready earlier? I recall we wanted `SafeBrowsingCheck` to be used inside `PageExperienceCheck` rather than the `ui/` directory, is there a clean way to update the progress UI with that structure? We don't have to do this now (polish is next week) but it will be good to make changes with this in mind.